packages/components/pie-webc-core/src/internals/PieElement.ts

```diff
@@ -0,0 +1,10 @@
+import { LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+
+export class PieElement extends LitElement {
+    @property({ type: String, reflect: true })
+    // TODO - Next PR will add a global declaration to allow us to remove this ts-ignore
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
The fix for this is easy but the PR size would be huge, so will do as a follow up immediately after this one is merged.